### PR TITLE
update(jiva): update vendor and use go modules 

### PR DESCRIPTION
**Pull request by utkarshmani1997**
- Update vendor with the latest stable version
- Remove trash and dapper
- Use go modules for dependency management
- Remove backup code since it's not getting used

Signed-off-by: Utkarsh Mani Tripathi <utkarshmani1997@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,7 @@ integration/.idea/
 integration/.tox/
 integration/.venv/
 integration/MANIFEST
+cscope.files
+cscope.out
+f1
+f2
```
Comment (prateekpandey14) [inline .gitignore]: is it required ?

File: cscope.files [hunk 2]
```diff
@@ -0,0 +1,693 @@
+/home/golearn/src/github.com/openebs/jiva/Makefile
+/home/golearn/src/github.com/openebs/jiva/app/replica.go
```
Comment (prateekpandey14) [inline cscope.files]: remove this file

File: libiscsi-testsuite/tags [hunk 3]
```diff
@@ -0,0 +1,6 @@
+!_TAG_FILE_FORMAT	2	/extended format; --format=1 will not append ;" to lines/
+!_TAG_FILE_SORTED	1	/0=unsorted, 1=sorted, 2=foldcase/
```
Comment (prateekpandey14) [inline libiscsi-testsuite/tags]: we can remove this one as well

File: .gitignore (hunk 1 above)
Comment (utkarshmani1997) [reply .gitignore]: yes, just to ignore the files used while integration testing and for other purpose

File: cscope.files (hunk 2 above)
Comment (utkarshmani1997) [reply cscope.files]: ok

File: libiscsi-testsuite/tags (hunk 3 above)
Comment (utkarshmani1997) [reply libiscsi-testsuite/tags]: ok

**Comment by vishnuitta** [comment]
cc: @AmitKumarDas can you please review makefile/go modules?

**Pull request merged**